### PR TITLE
feat: add spry framework

### PR DESCRIPTION
**Pull request by medz**
Hello, I added spry framework, It's a Dart web framework.

framework docs: https://spry.fun

### CODE REVIEW

File: dart/spry/config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+framework:
+  name: Spry
```
Comment (waghanza) [inline dart/spry/config.yaml:2]: not used

File: dart/spry/pubspec.lock
Comment (waghanza) [inline dart/spry/pubspec.lock:1]: could you remove this file ? we do not keep lockfiles here, to make sure the last patched version of framework is tested

File: dart/spry/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: spry_app
+
+environment:
+  sdk: '>=3.2.0 <3.3.0'
+
+dependencies:
+  spry: ^3.2.0
```
Comment (waghanza) [inline dart/spry/pubspec.yaml:7]: not remember exactly, but does it guarantee to be on 3.2.x?

**Review by waghanza — COMMENTED** [review]
Thanks for your contribution 🎉

File: dart/spry/config.yaml (hunk 1 above)
Comment (medz) [reply dart/spry/config.yaml]: i delete it

File: dart/spry/pubspec.lock
Comment (medz) [reply dart/spry/pubspec.lock:1]: i delete it

File: dart/spry/pubspec.yaml (hunk 2 above)
Comment (medz) [reply dart/spry/pubspec.yaml:7]: Yes, it is guaranteed to always work on 3.2.x

**Review by waghanza — APPROVED** [review]

**Pull request merged**